### PR TITLE
Created Contributors file

### PR DESCRIPTION
**Pull request by gauravsaini02**
I have added 2 files under Workflows directory.
1. Contributors.md : Explaining about the steps to be taken to contribute.
2. CODE_OF_CONDUCT.md : Explaining the ethics to be followed while contributing

### CODE REVIEW

File: .github/workflows/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Reports of abusive, harassing, or otherwise inappropriate behavior can be sent to info@mifos.org, the project team's email address. After each complaint is examined and looked into, a response that is judged essential and fitting for the situation will be given. The project team has a duty to keep the identity of the incident reported discreet. Specific enforcement policies may have additional information posted separately.
```
Comment (vkehfdl1) [inline .github/workflows/CODE_OF_CONDUCT.md:37]: You can change this e-mail address to 'jeffrey@markr.ai' and 'vkehfdl1@gmail.com'

File: .github/workflows/contributors.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+Remember that this is an inclusive community, committed to creating a safe, positive environment. See the whole [Code of Conduct](CODE_OF_CONDUCT.md) and please follow it in all your interactions with the project.
+
+
+## Submitting or Requesting an Issue/Enhancement
+
+### The best ways to report issues or make requests for improvements are as follows:
+- Please explore the issue tracker before submitting an issue. There may already be an issue for your issue, and the conversation may have made remedies readily apparent.
+- When creating the problem, adhere to the problem Template.
+- If there are any, including screenshots (especially for UI-related concerns).
+- Include mockups for UI improvements or procedures to obtain a clear understanding.
```
Comment (vkehfdl1) [inline .github/workflows/contributors.md:27]: We don't have UI yet, so delete 'UI improvements' at here.

File: .github/workflows/contributors.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+Remember that this is an inclusive community, committed to creating a safe, positive environment. See the whole [Code of Conduct](CODE_OF_CONDUCT.md) and please follow it in all your interactions with the project.
+
+
+## Submitting or Requesting an Issue/Enhancement
+
+### The best ways to report issues or make requests for improvements are as follows:
+- Please explore the issue tracker before submitting an issue. There may already be an issue for your issue, and the conversation may have made remedies readily apparent.
+- When creating the problem, adhere to the problem Template.
+- If there are any, including screenshots (especially for UI-related concerns).
```
Comment (vkehfdl1) [inline .github/workflows/contributors.md:26]: Do not need 
`(especially for UI-related concerns)`

**Review by vkehfdl1 — CHANGES_REQUESTED** [review]
Hi @gauravsaini02 Thanks for your first contribution.
Overall, it is great.
But, there are minor changes to be done.
Plus, two things need to be changed.

1. Move two .md files to root folder. I don't know why these file in .github/workflows directory.
2. Change contributors.md name to `CONTRIBUTING.md`

**Review by vkehfdl1 — APPROVED** [review]
LGTM

**Review by bwook00 — APPROVED** [review]
LGTM! Thanks for your first contribution 👍 👍

**Comment by gauravsaini02** [comment]
The checks have failed for my files. Kindly let me know the possible reason and its solution.

**Comment by vkehfdl1** [comment]
@gauravsaini02  We are figuring out how can we pass your test. 

One of the solution is to set your own API key at github secrets on your forked repository.

Go to forked repository > Settings > Secrets and Variables > Actions and set Repository Secrets.
You have to set OPENAI_API_KEY, JINAAI_API_KEY, and COHERE_API_KEY.

Sorry for the process. Me and @bwook00 will find a way that you don't have to set those variables. 
If you want to pass it quickly, please set the secrets. Or you can wait for the solution. Thank you.

**Comment by vkehfdl1** [comment]
@gauravsaini02 I send a PR for resolving test issue.
https://github.com/Marker-Inc-Korea/AutoRAG/pull/354
Please wait for this will be merged, and the next time your PR will be merged. Thanks

**Comment by gauravsaini02** [comment]
Hi Dear,
Thanks a lot....
I was looking forward to contribute to technical part in upcoming future. This issue will still create trouble i guess...

**Pull request merged**